src/cli/commands/telemetry/telemetry.ts

```diff
@@ -0,0 +1,57 @@
+import { logger } from "../../../core";
```

Comment:
shall we move the logger import after `import dotenv from "dotenv";`.. Idea is external ones we keep them together.. while internal relative references we keep together